README.md

```diff
@@ -36,15 +51,27 @@ def info_dump(customers)
   end
 end
 ```
+#--------------------#
+
+Snippet 3 - Big O: O(n)
+
+Reasoning: Snippet EX-A - Big O: O(n) because as the data set size grows the amount of loop iterations grows linearly. Also, "says that the algorithm's performance is directly proportional to the size of the data set being processed. Scanning an array takes O(N) time."
+
+It's looping through an array but only the first item within the array. So it speed seems like it would steadily proportional and linear.
+
```

Comment:
It's actually O(1) because no matter how large the data set is, we'll always just be loop at the very first element.